frontend/src/components/taskSelection/messages.js

```diff
@@ -283,9 +283,13 @@ export default defineMessages({
     id: 'project.tasks.sorting.id',
     defaultMessage: 'Sort by task number',
   },
-  sortByLastUpdate: {
-    id: 'project.tasks.sorting.date',
-    defaultMessage: 'Last updated first',
+  sortByMostRecentUpdate: {
+    id: 'project.tasks.sorting.taskdate',
```

Comment:
Could you modify it to `project.tasks.sorting.date`?

Reply:
@willemarcel  when i am changing it to back to 'project.tasks.sorting.date' then instead of "Mostly recently updated" "Last updated first" is showing

Reply:
maybe you need to run yarn build-locales to have the updated text.